pyvista/utilities/helpers.py

```diff
@@ -42,6 +42,11 @@ def vtk_bit_array_to_char(vtkarr_bint):
     return vtkarr
 
 
+def vtk_id_list_to_array(vtk_id_list):
+    """Convert a vtkIdList to a NumPy array."""
+    return np.array([vtk_id_list.GetId(i) for i in range(vtk_id_list.GetNumberOfIds())])
```

Comment:
Does anyone have ideas on how to speed this up/avoid the comprehension?

Reply:
I had a look at the python module numpy_support of vtk and I think that they use the function `numpy.frombuffer` to create the numpy object

https://github.com/Kitware/VTK/blob/c4442f8b87682560c99b737e1c027bcbf869e760/Wrapping/Python/vtkmodules/util/numpy_support.py#L201

Reply:
Unfortunately `frombuffer` doesn't work for `vtkIdList`, only for data arrays. I wonder if there's a clean way to just convert a `vtkIdList` to `vtkIntArray`?